res/database/Default/database_programmes_fictional.xml

```diff
@@ -1178,60 +1178,79 @@ Do the two offer enough material for an exciting panel?</en>
+						<de>Es bleibt Prinz [2|First] nicht verborgen, dass sich seine Frau im Umgang mit den Medien wesentlich geschickter erweist als er selbst. Ihre kurzfristigen Triumphe genügen [3|First] jedoch nicht.</de>
+						<en>It does not go unnoticed by Prince [2|First] that his wife is proving to be much more adept at dealing with the media than he is. However, her short-term triumphs are not enough for [3|First].</en>
+					</description>
 					<ratings critics="42" speed="56" />
 				</programme>
 				<programme id="558991e0-7971-4489-912b-c7ab299c5062" product="7" licence_type="2" tmdb_id="0" imdb_id="" creator="5578" created_by="Ronny">
 					<title>
-						<de>Dynamit - Das Magazin Folge 2</de>
-						<en>Dynamite - The Magazine Episode 2</en>
+						<de>Dynamit [2|Nick] und [3|Nick] Spezial - Folge 2</de>
+						<en>Dynamite [2|Nick] and [3|Nick] Special - The Magazine Episode 2</en>
 					</title>
+					<description>
+						<de>Gerüchten zufolge kriselt es in der Ehe, und [3|First] möchte nicht, dass die Öffentlichkeit sie für ein Scheitern ihrer Ehe verantwortlich machen würde.</de>
+						<en>Rumor has it that there is a crisis in the marriage, and [3|First] does not want the public to hold her responsible for a breakup of their marriage.</en>
+					</description>
 					<ratings critics="28" speed="51" />
 				</programme>
 				<programme id="a45a7dd5-af5b-4365-80c5-4cf6d7c441c3" product="7" licence_type="2" tmdb_id="0" imdb_id="" creator="5578" created_by="Ronny">
 					<title>
-						<de>Dynamit - Das Magazin Folge 3</de>
-						<en>Dynamite - The Magazine Episode 3</en>
+						<de>Dynamit [2|Nick] und [3|Nick] Spezial - Folge 3</de>
+						<en>Dynamite [2|Nick] and [3|Nick] Special - The Magazine Episode 3</en>
 					</title>
+					<description>
+						<de>Weiter heißt es, [3|First] denke, es sei aber ein Fehler, weiterhin Stillschweigen über den Zustand ihrer Ehe zu bewahren. Sie trifft sich angeblich mit einem guten Bekannten und Journalisten, um an einem Buch zu schreiben.</de>
+						<en>It is further said that [3|First] thinks it would be a mistake to continue to keep quiet about the state of her marriage, however. She is reportedly meeting with a close acquaintance and journalist to write a book.</en>
+					</description>
 					<ratings critics="70" speed="42" />
 				</programme>
 				<programme id="d8f1c57b-feac-483e-9689-80252d487126" product="7" licence_type="2" tmdb_id="0" imdb_id="" creator="5578" created_by="Ronny">
 					<title>
-						<de>Dynamit - Das Magazin Folge 4</de>
-						<en>Dynamite - The Magazine Episode 4</en>
+						<de>Dynamit [2|Nick] und [3|Nick] Spezial - Folge 4</de>
+						<en>Dynamite [2|Nick] and [3|Nick] Special - The Magazine Episode 4</en>
 					</title>
+					<description>
+						<de>Heiße exklusive Insider-Informationen nur hier! [2|First] hat ja angeblich eine Affäre, und [3|First] hat Beweise dafür aus seiner Aktentasche gestohlen, die dann in ihrem Buch veröffentlicht würden.</de>
```

Comment:
"Heiße exklusive Insider-Informationen nur hier! [2|First] hat ja angeblich eine Affäre, und [3|First] hat Beweise dafür aus seiner Aktentasche gestohlen, die dann in ihrem Buch veröffentlicht würden."

That phrase sounds ... a bit odd. "ja" can be "removed" and even the second "hat" is superfluous (even if you think it isn't).

Variations:
Nur hier erhältst du exklusive Insider-Informationen! Gerüchten zufolge soll [2|First] eine Affäre haben und [3|First] soll Beweise dafür in seiner Aktentasche haben, die angeblich in ihrem Buch veröffentlicht werden.

Exklusive Insider-Infos nur hier: [2|First] soll eine Affäre haben und [3|First] hat Beweise aus seiner Aktentasche gestohlen, die in einem Buch veröffentlicht würden.

Erhalte exklusive Insider-Infos: [2|First] hat angeblich Affäre, [3|First] stahl Beweise aus Aktentasche für Buchveröffentlichung. Nur hier!

Erhalte exklusive Insider-Infos: [2|First] soll Affäre haben und [3|First] hat angeblich Beweise in der Tasche, die in einem Buch veröffentlicht werden sollen. Nur hier erfährst du mehr!

Reply:
I compiled all this from a few sentences from wikipedia and didn't do any deeper research, and from the article there it's not clear what was definitive known fact in July 1991, what was rumours, and what was unknown yet, but I *think* at that time it was already kinda known that Charles had an affair, which I wanted to emphasize with "ja". `:)`

Reply:
First part should read better now, but I kept the second part more or less.

Reply:
The problem is the "style" in "hat ja angeblich". This no "yellow press" writing style. This is what they _say_ during dubbing the video material (if they do it "personal style"). But "written" (eg in the "Bauchbinde"/inserts) it looks a bit off.

Anyways, am as usual just nitpicking :D